src/UriInterface.php

```diff
@@ -18,6 +18,78 @@
 interface UriInterface
 {
 	/**
+	 * Include the scheme (http, https, etc.)
+	 *
+	 * @var    integer
+	 * @since  __DEPLOY_VERSION__
+	 */
+	const SCHEME = 1;
+
+	/**
+	 * Include the user
+	 *
+	 * @var    integer
+	 * @since  __DEPLOY_VERSION__
+	 */
+	const USER = 2;
+
+	/**
+	 * Include the password
+	 *
+	 * @var    integer
+	 * @since  __DEPLOY_VERSION__
+	 */
+	const PASS = 4;
+
+	/**
+	 * Include the host
+	 *
+	 * @var    integer
+	 * @since  __DEPLOY_VERSION__
+	 */
+	const HOST = 8;
+
+	/**
+	 * Include the port
+	 *
+	 * @var    integer
+	 * @since  __DEPLOY_VERSION__
+	 */
+	const PORT = 16;
+
+	/**
+	 * Include the path
+	 *
+	 * @var    integer
+	 * @since  __DEPLOY_VERSION__
+	 */
+	const PATH = 32;
+
+	/**
+	 * Include the query string
+	 *
+	 * @var    integer
+	 * @since  __DEPLOY_VERSION__
+	 */
+	const QUERY = 64;
+
+	/**
+	 * Include the fragment
+	 *
+	 * @var    integer
+	 * @since  __DEPLOY_VERSION__
+	 */
+	const FRAGMENT = 128;
+
+	/**
+	 * Include all available url parts (scheme, user, pass, host, port, path, query, fragment)
+	 *
+	 * @var    integer
+	 * @since  __DEPLOY_VERSION__
+	 */
+	const ALL = 255;
```

Comment:
Yes. I was being stupid. 255 is a combination of all the above. Not a new value. Mea culpa